src/cui.cpp

```diff
@@ -175,7 +215,7 @@ void Line::show(int row, unsigned int proglen) {
   mvaddstr_truncate_trailing(row, column_offset_user, username.c_str(),
                              username.size(), COLUMN_WIDTH_USER);
 
-  mvaddstr_truncate_leading(row, column_offset_program, m_name, strlen(m_name),
+  mvaddstr_truncate_cmdline(row, column_offset_program, m_name, m_cmdline,
```

Comment:
After this change `mvaddstr_truncate_leading` is no longer used at all - just remove it?